src/lib/vocab/role/role.schema.ts

```diff
@@ -1,6 +1,11 @@
 export const RoleSchema = {
 	$id: '/schemas/Role.json',
 	type: 'object',
+	description: `
+		Roles exist within the context of a single Community.
+		They have Policies associated with them that allow for actions to be taken within the system.
+		When a Persona is Assigned a Role, that actor may take any action allowed by the Roles policies.
```

Comment:
```suggestion
		When a Persona is has a Role via an Assignment, that actor may take any action allowed by the Role's Policies.
```

or if instead you want to keep `Assigned` I think lowercasing is better